index.d.ts

```diff
@@ -74,6 +86,21 @@ declare module "replicate" {
     predictionId: string;
   }
 
+  export interface TrainingsCreateOptions {
+    destination: string;
+    input: any;
+    webhook?: string;
+    webhook_events_filter?: WebhookEventType[];
+  }
+
+  export interface TrainingsGetOptions {
+    trainingId: string;
+  }
+
+  export interface TrainingsCancelOptions {
+    trainingId: string;
+  }
```

Comment:
These also don't match. We're passing in a `number` in the test suite, not an options object with a `string` property.